docs/source/contents/4_draft_mapping.rst

```diff
@@ -1662,6 +1662,407 @@ opaque:hostItem
+
+    <location>
+        <physicalLocation>University of Memphis. Special Collections</physicalLocation>
+        <holdingSimple>
+            <copyInformation>
+                <shelfLocator>Manuscript Number 5</shelfLocator>
+            </copyInformation>
+        </holdingSimple>
+        <holdingExternal>
+            <holding xsi:schemaLocation="info:ofi/fmt:xml:xsd:iso20775 http://www.loc.gov/standards/iso20775/N130_ISOholdings_v6_1.xsd">
+                <physicalAddress>
+                    <text>City: Memphis</text>
+                    <text>County: Shelby County</text>
+                    <text>State: Tennessee</text>
+                </physicalAddress>
+            </holding>
+        </holdingExternal>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+    @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+
+    <https://example.org/objects/1>
+        relators:rps "University of Memphis. Special Collections" ;
+        skos:note "Shelf locator: Manuscript Number 5" .
+
+physicalLocation with @displayLabel="Address"
+---------------------------------------------
+
+Use Case
+^^^^^^^^
+
+Some of items with the physicalLocation of Pi Beta Phi Fraternity also have a physicalLocation subelement with the displayLabel attribute of Address.
+
+Justification
+^^^^^^^^^^^^^
+
+Similar to the holdingExternal, we will opt drop this information to maintain simplicity of our data from a technical standpoint.
+
+Xpath
+^^^^^
+
+`mods:location/mods:physicalLocation[@displayLabel=Address]`
+
+Decision
+^^^^^^^^
+We will drop all information for `holdingExternal`.
+
+`Example record from volvoices:2199 <https://digital.lib.utk.edu/collections/islandora/object/volvoices:2199/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation>Pi Beta Phi Fraternity</physicalLocation>
+        <physicalLocation displayLabel="Address">1154 Town and Country Commons Drive, Town and Country, Missouri 63017</physicalLocation>
+        <shelfLocator>Box 36, Folder 14</shelfLocator>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+    @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+
+    <https://example.org/objects/1>
+        relators:rps "Pi Beta Phi Fraternity" ;
+        skos:note "Shelf locator: Box 36, Folder 14" .
+
+physicalLocation with @displayLabel="Collection"
+------------------------------------------------
+
+Use Case
+^^^^^^^^
+In a few of collections for Arrowmont, we will find items having a physicalLocation subelement with the displayLabel attribute of Collection, along with Detailed Location, City and State.
+
+Justification
+^^^^^^^^^^^^^
+Because these records do not already have a dbo:collection predicate, we will transcribe the string literal to dbo:collection for `physicalLocation[@displayLabel=Collection]`. No other data here needs to be retained and will be dropped.
```

Comment:
If you use back ticks, treat it as inline code.  Also, xpath is wrong.  Switch to:

```
Because these records do not already have a dbo:collection predicate, we will transcribe the string literal to dbo:collection for :code:`physicalLocation[@displayLabel="Collection"]`. No other data here needs to be retained and will be dropped.
```